test/parallel/test-process-versions.js

```diff
@@ -9,4 +9,8 @@ if (common.hasCrypto) {
   expected_keys.push('openssl');
 }
 
+if (process.config.variables.v8_enable_i18n_support !== undefined) {
```

Comment:
This is arguably wrong because the variable can be defined but zero.  Maybe check that `typeof Intl !== 'undefined'`?